homeassistant/config.py

```diff
@@ -247,12 +247,12 @@ def _filter_bad_internal_external_urls(conf: dict) -> dict:
 
 def _raise_issue_if_historic_currency(hass: HomeAssistant, currency: str) -> None:
     if currency not in HISTORIC_CURRENCIES:
-        ir.async_delete_issue(hass, "homeassistant", "historic_currency")
+        ir.async_delete_issue(hass, CONF_CORE, "historic_currency")
```

Comment:
I think we should fix this and rename the import to something other than `CONF_CORE`. We only prefix config key string constants with `CONF_`. That's not what this is.

Reply:
We also reference this as `conf_util.CONF_CORE`. I would suggest to use `DOMAIN_HA` or `DOMAIN_HOMEASSISTANT` as an alias name for the `homeassistant` domain.

Reply:
Created a new local const in `config.py` with name `DOMAIN_HA` to be used to access the issue registry.

Reply:
Why do we make a new constant? Normally we just do:
```
from homeassistant.core import DOMAIN as HA_DOMAIN
```

Reply:
Have renamed all occurences of `CONF_CORE`